examples/express-ts/package.json

```diff
@@ -14,24 +14,24 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.16.1",
-    "express": "^4.14.1",
-    "express-promise-router": "^1.1.1",
-    "knex": "^0.12.7",
+    "body-parser": "^1.17.2",
+    "express": "^4.15.3",
+    "express-promise-router": "^2.0.0",
+    "knex": "^0.13.0",
     "lodash": "^4.17.4",
-    "morgan": "^1.8.1",
-    "objection": "^0.7.0",
+    "morgan": "^1.8.2",
+    "objection": "../..",
```

Comment:
> Sure, but we would need to write tests for the example projects

I think that might be a good effort--I'm always nervous/suspicious of untested code (and it would provide some examples to people on how to test code that uses Objection).

Do you want some PRs that add tests? I'd like to see how you guys handle testing, in any event...